### PR TITLE
Update Van der Pol benchmark

### PR DESCRIPTION
**Pull request by mforets**
- [x] add benchmark for mu = 2
- [x] add function to split initial state
- [x] add plot for mu = 2

For later: how to pass mu to the taylorized function as a parameter?

---

CC: @lbenet

### CODE REVIEW

**Comment by mforets** [comment]
Example of splitting the initial set for mu = 2:

```julia
X0_μ2 = Hyperrectangle(low=[1.55, 2.35], high=[1.85, 2.45])

plot(split(X0_μ2, 7, 1)[:], alpha=.5)
plot!(X0_μ2, alpha=.2, linewidth=4.)

```

Screen Shot 2019-03-29 at 13 39 20

**Comment by mforets** [comment]
Result using splitting in the "x" directions, satisfies the property:

```julia
# ===============
# CASE mu = 2
# ===============

X0_μ2 = Hyperrectangle(low=[1.55, 2.35], high=[1.85, 2.45])

# algorithm-specific options
𝑂jets = Options(:abs_tol=>1e-10, :orderT=>8, :orderQ=>2, :max_steps=>500)

sol_2 = []

for X0i in split(X0_μ2, 8, 1)
    𝑃, 𝑂 = vanderpol(μ=2, T=8.0,
                     X0=X0i,
                     property=(t,x) -> x[2] < 4.0)

    sol_2_i = solve(𝑃, 𝑂, op=TMJets(𝑂jets))
    push!(sol_2, sol_2_i)
    @assert all([ρ([0.0, 1.0], sol_2_i.Xk[i].X) < 4.0 for i in eachindex(sol_2_i.Xk)])
end

plot(sol_2[1])
plot!(sol_2[2])
plot!(sol_2[3])
plot!(sol_2[4])
```

Screen Shot 2019-03-29 at 13 58 57

**Comment by mforets** [comment]
plot for mu=1  (red) *and* mu=2 (blue)

Screen Shot 2019-03-29 at 15 41 47

File: models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl [hunk 1]
```diff
@@ -32,22 +32,34 @@ SUITE["VanDerPol"]["μ = 1: x[2] <= 2.75"] = @benchmarkable solve($𝑃, $𝑂,
...
+function compute_μ2(;n::Int=8, validate=true)
+    sol_2 = []
+    for X0i in split(X0_μ2, n, 1)
+        𝑃, 𝑂 = vanderpol(μ=2.0, T=8.0, X0=X0i, property=(t,x) -> x[2] < 4.0)
+        𝑂jets = Options(:abs_tol=>1e-10, :orderT=>8, :orderQ=>1, :max_steps=>500)
+        sol_2_i = solve(𝑃, 𝑂, op=TMJets(𝑂jets))
+        push!(sol_2, sol_2_i)
+        if validate
+            # verify that specification holds
+            @assert all([ρ([0.0, 1.0], sol_2_i.Xk[i].X) < 4.0 for i in eachindex(sol_2_i.Xk)])
```
Comment (schillic) [inline models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl]: Why do we need this? The function already checks the property and throws an error if the property does not hold.

File: models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl [hunk 2]
```diff
@@ -32,22 +32,34 @@ SUITE["VanDerPol"]["μ = 1: x[2] <= 2.75"] = @benchmarkable solve($𝑃, $𝑂,
...
-@assert all([ρ([0.0, 1.0], sol_2.Xk[i].X) < 4.0 for i in eachindex(sol_2.Xk)])
+𝑂jets = Options(:abs_tol=>1e-10, :orderT=>8, :orderQ=>2, :max_steps=>500)
+
+# the idea is to split the initial states along the "x" direction
+# n controls the splitting
+nsplits_x = 8
+
+function compute_μ2(;n::Int=8, validate=true)
+    sol_2 = []
+    for X0i in split(X0_μ2, n, 1)
```
Comment (schillic) [inline models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl]: We can use the new function `partition` in `LazySets`.

File: models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl (hunk 1 above)
Comment (mforets) [reply models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl]: that's true. we also have an online check in BFFPSV18. i've added this check here and in the AFF just for a second "sanity checking".

Comment (schillic) [reply models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl]: But it will count in the benchmark time...

Comment (mforets) [reply models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl]: For benchark time there is the flag `validate=false`.

File: models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl (hunk 2 above)
Comment (mforets) [reply models/ARCH/NLN/VanDerPol/vanderpol_benchmark.jl]: Ill add it in a further review.

**Pull request merged**